### PR TITLE
Ddd

### PR DESCRIPTION
**Pull request by OleksandRomaniuk**

### CODE REVIEW

**Review by ajax-gnatiuk-s — CHANGES_REQUESTED** [review]

File: web/group-chat-subdomain/src/main/kotlin/com/example/ajax/application/GroupChatServiceInPort.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.example.ajax.application
```
[the remark this answers is not in the document]
Comment (OleksandRomaniuk) [reply web/group-chat-subdomain/src/main/kotlin/com/example/ajax/application/GroupChatServiceInPort.kt:1]: Also mooved

File: web/user-subdomain/src/main/kotlin/com/example/ajax/infrastructure/nats/NatsConfiguration.kt [hunk 2]
```diff
@@ -11,7 +11,7 @@ import reactor.core.scheduler.Schedulers
 @Configuration
 class NatsConfiguration {
     @Bean
-    fun natsConnection(@Value("\${nats.url}") natsUrl: String): Connection = Nats.connect(natsUrl)
+    fun natsConnection(@Value("nats://localhost:4222") natsUrl: String): Connection = Nats.connect(natsUrl)
```
[the remark this answers is not in the document]
Comment (OleksandRomaniuk) [reply web/user-subdomain/src/main/kotlin/com/example/ajax/infrastructure/nats/NatsConfiguration.kt:14]: did it

File: web/private-chat-subdomain/src/main/kotlin/com/example/ajax/application/port/CacheableRepository.kt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package com.example.ajaxproject.repository.cacheable
+package com.example.ajax.application.port
```
[the remark this answers is not in the document]
Comment (OleksandRomaniuk) [reply web/private-chat-subdomain/src/main/kotlin/com/example/ajax/application/port/CacheableRepository.kt:1]: But if i move it to infrastructure - i will use infrastructure in application. maybe just rename it adding OutPort?

File: web/user-subdomain/src/main/kotlin/com/example/ajax/application/svc/UserService.kt [hunk 4]
```diff
@@ -0,0 +1,60 @@
+package com.example.ajax.application.svc
```
[the remark this answers is not in the document]
Comment (ajax-gnatiuk-s) [reply web/user-subdomain/src/main/kotlin/com/example/ajax/application/svc/UserService.kt:1]: please, don't resolve comment until you commit changes

File: web/private-chat-subdomain/src/main/kotlin/com/example/ajax/infrastructure/entity/RedisPrivateChatRoom.kt [hunk 5]
```diff
@@ -1,4 +1,4 @@
-package com.example.ajax.infrastructure.redis.entity
+package com.example.ajax.infrastructure.entity
```
Comment (ajax-gnatiuk-s) [inline web/private-chat-subdomain/src/main/kotlin/com/example/ajax/infrastructure/entity/RedisPrivateChatRoom.kt:1]: the previous version was better. In package `*.infrastructure.entity`

Comment (OleksandRomaniuk) [reply web/private-chat-subdomain/src/main/kotlin/com/example/ajax/infrastructure/entity/RedisPrivateChatRoom.kt]: Agree

**Pull request merged**